tests/prescribed_velocity/screen-output

```diff
@@ -0,0 +1,40 @@
+-----------------------------------------------------------------------------
+-- This is ASPECT, the Advanced Solver for Problems in Earth's ConvecTion.
+--     . version 1.4.pre
+--     . running in DEBUG mode
+--     . running with 1 MPI process
+--     . using Trilinos
+-----------------------------------------------------------------------------
+
+Loading shared library <./libprescribed_velocity.so>
+
+Number of active cells: 64 (on 4 levels)
+Number of degrees of freedom: 948 (578+81+289)
+
+*** Timestep 0:  t=0 years
+   Solving temperature system... 0 iterations.
+   Rebuilding Stokes preconditioner...
+   Solving Stokes system... 30+6 iterations.
+
+   Postprocessing:
+     RMS, max velocity: 3.27e+07 m/year, 4.6e+07 m/year
```

Comment:
Yes, this is fine. The velocities are prescribed to ~1 m/s which corresponds to ~3m/yr. One needs to be careful here, as everywhere, to make sure they're using consistent units throughout their models.